mRides-app/Resources/layout/layout2.axml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="horizontal"
```

Comment:
@nim442  Layout2 was mainly for test purposes (designer view) and layout 1 i scoped it out from the project (it was supposed to go to git ignore).  Ideally, Layout 2 is the same thing as bottombar.xml but the merge tag is not there.